Sources/StreamChat/Config/StreamRuntimeCheck.swift

```diff
@@ -31,4 +31,9 @@ public enum StreamRuntimeCheck {
     ///
     /// Enables reusing unchanged converted items in database observers.
     public static var _isDatabaseObserverItemReusingEnabled = true
+    
+    /// For *internal use* only
+    ///
+    /// Uses version 2 for offline state sync.
+    public static var _isSyncV2Enabled = true
```

Comment:
We should also have a ticket to remove this in 2-3 releases and only support V2.

Reply:
Added a [ticket](https://stream-io.atlassian.net/browse/PBE-5533)